task9/persons.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
This file shouldn't be committed as it's part of the runtime of the application running. Please remove it and add to gitignore too